### ISSUE TITLE
Avatar/Cover Image Specifications

### ISSUE THREAD

**Issue by Phenomephrene** [MbinOrg/mbin#1014]
QoL/UX Improvement Request:

If possible, on user settings/profile page include supported file types and image size requirements next to or near the area to change Avatar and Cover images. This may reduce any frustration in trying to upload appropriate images, and increase feature adoption.

Thanks for considering.

**Comment by nobodyatroot** [comment]
While I agree this is a necessary enhancement, please use the templates when submitting issues so we can deduce if there's a problem with a specific release and/or instance running said release. That said, there's a high priority bug that's impacting images in the latest tagged 1.7.1-rc1 release of Mbin and your instance could be impacted if they are not running the latest code from the main branch. https://github.com/MbinOrg/mbin/pull/1011

**Comment by Phenomephrene** [comment]
Sorry about that. I'm new to github, and probably one of the less tech savvy fediverse users you're likely to come across.

My home instance is thebrainbin.org which says it is currently running mbin version 1.7.1-rc1

For what it's worth, the current behavior exactly mirrors the types of issues I was having when based out of kbin.social. Attempting different formats (e.g. jpegs, pngs) of various different file sizes, where even trying to match format and image size of images that have previously worked won't necessarily guarantee that a new image with successfully update. I figured it was still something I was doing incorrectly since, like I said, I'm not the most tech savvy person.

If this is due to the known bug or not, clarifying what file type/size _should_ work seemed maybe an easy addition (?) and is useful info to make on-boarding new members a bit friendlier.

Thanks again.

**Comment by nobodyatroot** [comment]
> Sorry about that. I'm new to github, and probably one of the less tech savvy fediverse users you're likely to come across.
> 
> My home instance is thebrainbin.org which says it is currently running mbin version 1.7.1-rc1
> 
> For what it's worth, the current behavior exactly mirrors the types of issues I was having when based out of kbin.social. Attempting different formats (e.g. jpegs, pngs) of various different file sizes, where even trying to match format and image size of images that have previously worked won't necessarily guarantee that a new image will successfully update. I figured it was still something I was doing incorrectly since, like I said, I'm not the most tech savvy person.
> 
> If this is due to the known bug or not, clarifying what file type/size _should_ work seemed maybe an easy addition (?) and is useful info to make on-boarding new members a bit friendlier.
> 
> Thanks again.

sounds good, and thanks for clarify. i believe @BentiGorlich has thebrainbin.org updated to include the fix, so probably not related :-)

**Comment by BentiGorlich** [comment]
Yeah thebrainbin only runs on releases. Thats what I decided. Though I can make an exception for this one and just pull main, so image uploads can work again, though I think just setting the env var for max kmage size already fixes it right? It just defaults to 0 bytes max size

**Comment by nobodyatroot** [comment]
> Yeah thebrainbin only runs on releases. Thats what I decided. Though I can make an exception for this one and just pull main, so image uploads can work again, though I think just setting the env var for max kmage size already fixes it right? It just defaults to 0 bytes max size

the bug causes the default to not load correctly, so thats fair if you had it set to something in the `.env`... i think.

**Comment by BentiGorlich** [comment]
I updated thebrainbin, so your problems with image uploads should be fixed. However I agree that the max image size should be communicated in the UI

**Comment by Phenomephrene** [comment]
Great! Successful upload on my end. I appreciate your effort! Thanks.

**Comment by BentiGorlich** [comment]
No problem :)

**Comment by BentiGorlich** [comment]
I think this one is done with #1019 
or do you think the supported formats are necessary?
